libnetdata/config/appconfig.c

```diff
@@ -280,6 +280,54 @@ long long appconfig_get_number(struct config *root, const char *section, const c
     return strtoll(s, NULL, 0);
 }
 
+int appconfig_get_duration(struct config *root, const char *section, const char *name, char * value)
```

Comment:
Let's not have the exact same code as health_parse_duration copied. Please have a common function that can be called both from here and from health_config.c

Reply:
Put `parse_duration` in `appconfig.c` as well, at least that way we have things in two files instead of three.

Reply:
Thought I replied but I can't see it here, better put `parse_duration` in `appconfig.c` as well. At least we only use two files this way.